app/models/attachment.js

```diff
@@ -317,6 +324,38 @@ export function addModel(dbAdapter) {
       }
     }
 
+    async sanitizeMetadata(filePath) {
+      // Synthetic and permanent tags
+      const nonWritableTags = ['GPSPosition', 'SerialNumberFormat'];
+
+      // try {
+      const tags = await exiftool.read(filePath);
+      const tagsToClean = {};
+
+      for (const tag of Object.keys(tags)) {
+        if (
+          (/GPS/i.test(tag) || /Serial/i.test(tag) || /Owner/i.test(tag)) &&
```

Comment:
Should we also clean `UserComment`, `MakerNote*`, `ApplicationNotes`?

Reply:
I don't really know what the tags mean or how unsafe it is to keep them.